Source/Lib/Codec/EbEncHandle.c

```diff
@@ -3958,10 +3953,8 @@ EB_ERRORTYPE EbOutputBufferHeaderCreator(
     }
 
 	// Initialize Header
+    // pBuffer is allocated in PK
```

Comment:
is spacing correct?

Reply:
Mine is correct. The lines around is tab. Tab will be replaced by space in #302.